tests/Unit/ActionTesting.hpp

```diff
@@ -20,11 +20,37 @@
 #include "Utilities/Gsl.hpp"
 #include "Utilities/NoSuchType.hpp"
 #include "Utilities/PrettyType.hpp"
+#include "Utilities/Requires.hpp"
 #include "Utilities/StdHelpers.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
+#include "Utilities/TypeTraits.hpp"
 
 namespace ActionTesting {
+namespace detail {
+template <typename T, typename = cpp17::void_t<>>
+struct get_mocking_list {
```

Comment:
This small chunk here is a very good example of seeing SFINAE in action, I would add it to the [dev guide](https://spectre-code.org/template_metaprogramming.html) :)